sentry_sdk/integrations/anthropic.py

```diff
@@ -46,7 +45,10 @@ def setup_once():
         if version < (0, 16):
             raise DidNotEnable("anthropic 0.16 or newer required.")
 
-        Messages.create = _wrap_message_create(Messages.create)
+        if version >= (0, 27, 0):
+            Messages.create = _wrap_message_create(Messages.create)
+        else:
+            Messages.create = _wrap_message_create_old(Messages.create)
```

Comment:
This is probably unnecessary but I'm not sure how the maintainers feel about separating major changes in Anthropic python sdk.

Reply:
I took the liberty to commit to your branch. When we only handle the "text" responses then it works the same in old and new Anthropic so we do not need this if (and the _wrap_message_create_old function)